prow/cmd/autobump/bump.sh

```diff
@@ -200,12 +197,13 @@ list() {
 
 split_on_commas() {
   local IFS=,
-  local array=("$1")
+  local array
+  read -ra array <<< "$1"
   echo "${array[@]}"
 }
 
 add_suffix() {
-  local array=("$1")
+  local array=($1)
```

Comment:
Why no quotes around the expansion here?

Reply:
Because I ran a local test and it didn't work
```
split_on_commas() {
  local IFS=,
  local array
  read -ra array <<< "$1"
  echo "${array[@]}"
}

add_suffix() {
  local array=("$1")
  local suffix="${2:-/*.yaml}"
  echo "${array[@]/%/$suffix}"
}

COMPONENT_FILE_DIR="dir1,dir2"
add_suffix "$(split_on_commas "${COMPONENT_FILE_DIR}")"
```

The output is:
```
dir1 dir2/*.yaml
```

Is it expected to be?
```
dir1/*.yaml dir2/*.yaml
```

Reply:
Is that because you changed `split_on_commas`? I don't know enough about bash to understand the effect of your changes.

Reply:
@kubernetes/bash-firefighters